rpi-imager-haos.json

```
@@ -45,9 +45,25 @@
         "pi3-64bit"
       ]
     },
+    {
+      "name": "Home Assistant OS 14.0 (HA Yellow)",
+      "description": "Home Assistant OS for Home Assistant Yellow. To be flashed directly to an NVMe drive or Yellow booted using rpiboot.",
```

Not sure I would use booted in the context of rpiboot, it might be confusing. The NVMe actually shows up too when using the MSD thingy. So I think we can drop the rpiboot hint here in general, e.g.:

```suggestion
      "description": "Home Assistant OS for Home Assistant Yellow. To be flashed directly to an NVMe drive or the Compute Module on Yellow  (using rpiboot).",
```